### PR TITLE
Completed css-four-column-layout.

### PR DESCRIPTION
**Pull request by KrisMackenroth**
Screenshot (366)
Screenshot (365)
Screenshot (364)
Screenshot (363)
Screenshot (362)

### CODE REVIEW

**Comment by KrisMackenroth** [comment]
Oops forgot to change some text.

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <div class="column-full">
+          <h1><b>How it Works</b></h1>
+        </div>
+
+      </div>
+
+
+
+
+
```
Comment (Zhiwutian) [inline css-four-column-layout/index.html:30]: make sure to remove any whitespace like this from the code. HTML should not have gaps in it.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Just need to remove the whitespace from the HTML

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**